docs/development/release.md

```diff
@@ -80,7 +79,7 @@ versions to that.
 Manually create a release on github & upload, but soon we'll publish shipbot which automates this...
```

Comment:
It looks like we're using shipbot. Are we not using it in all of it's glory? Is this ready to be updated?